### DISCUSSION TITLE
Ajuda /  help

### DISCUSSION THREAD

**Discussion by eriks306014** [community/community#31284] [Other Feature Feedback, Questions, & Ideas]
Boa noite por favor alguém pode me dizer se é possível criar várias páginas em um só Domínio aqui no github ?

Good evening, can anyone tell me if it is possible to create multiple pages in a single Domain here on github?

**Comment by Gustavonn07** [comment]
1.  Você pode criar as páginas com o próprio HTML usando o  `link` ou `a`.
2.  Você pode criar usando o javascript trocando de HTML e CSS ao clicar em um botão.
3.  Você pode criar páginas com repositórios diferentes e apenas linkar ele.

Mas sim, é possível! (não sei se ainda vai ser útil dado a diferença de tempo da pergunta)
